fmn/rules/utils.py

```diff
@@ -162,6 +235,27 @@ def invalidate_cache_for(config, fn, arg):
     return _cache.delete(key)
 
 
+def _get_packages_for(config, username, flags):
```

Comment:
They are.. opposites of each other?  One gets packagers for a package - the other gets packages for a packager.